src/core/Akka.API.Tests/CoreAPISpec.ApproveClusterTools.approved.txt

```diff
@@ -642,39 +642,37 @@ namespace Akka.Cluster.PubSub.Serializers.Proto
 }
 namespace Akka.Cluster.Tools.Client
 {
-    public class ClusterClient : Akka.Actor.ActorBase
+    public sealed class ClusterClient : Akka.Actor.ActorBase
     {
-        public readonly Akka.Cluster.Tools.Client.ClusterClientSettings Settings;
         public ClusterClient(Akka.Cluster.Tools.Client.ClusterClientSettings settings) { }
-        public Akka.Event.ILoggingAdapter Log { get; }
         protected override void PostStop() { }
         public static Akka.Actor.Props Props(Akka.Cluster.Tools.Client.ClusterClientSettings settings) { }
         protected override bool Receive(object message) { }
         public sealed class Publish
         {
-            public readonly object Message;
-            public readonly string Topic;
             public Publish(string topic, object message) { }
+            public object Message { get; }
+            public string Topic { get; }
         }
         public sealed class Send
         {
-            public readonly bool LocalAffinity;
-            public readonly object Message;
-            public readonly string Path;
             public Send(string path, object message, bool localAffinity = False) { }
+            public bool LocalAffinity { get; }
+            public object Message { get; }
+            public string Path { get; }
         }
         public sealed class SendToAll
         {
-            public readonly object Message;
-            public readonly string Path;
             public SendToAll(string path, object message) { }
+            public object Message { get; }
+            public string Path { get; }
         }
     }
-    public class ClusterClientReceptionist : Akka.Actor.IExtension
+    public sealed class ClusterClientReceptionist : Akka.Actor.IExtension
     {
         public ClusterClientReceptionist(Akka.Actor.ExtendedActorSystem system) { }
         public bool IsTerminated { get; }
-        public Akka.Actor.IActorRef PubSubMediator { get; }
+        public Akka.Actor.IActorRef Underlying { get; }
```

Comment:
Breaking change here, but that makes sense given that we're formalizing the API prior to RTM-ing it.